Fitness/Controllers/HabitTrackingController.swift

```diff
@@ -247,7 +254,7 @@ class HabitTrackingController: UIViewController {
         habitTableView.snp.makeConstraints { make in
             make.leading.trailing.equalToSuperview()
             make.top.equalTo(descriptionLabel.snp.bottom).offset(8)
-            let height = 68 + HabitTrackerOnboardingCell.height * ( habits.count + 1 )
+            let height = 84 + HabitTrackerOnboardingCell.height * ( habits.count + 1 )
```

Comment:
Can we bring this constant up to an enum `FileConstants` as a static let assignment. That way we can reuse it in `updateTableviewConstraints()` function as well.

Reply:
👍🏻